lib/compress/zstd_cwksp.h

```diff
@@ -275,6 +378,29 @@ MEM_STATIC void* ZSTD_cwksp_reserve_internal(
     return alloc;
 }
 
+/* Performs any final internal actions on the wksp before it is ready to go. */
+MEM_STATIC void ZSTD_cwksp_finalize(ZSTD_cwksp* ws) {
+    DEBUGLOG(5, "cwksp: finalizing: %zu table alignment bytes and %zu aligned alignment bytes",
+             ws->tableExtraAlignmentBytes, ws->alignedExtraAlignmentBytes);
+    if (ws->tableExtraAlignmentBytes == ZSTD_CWKSP_ALIGNMENT_BYTES) {
+        /* Due to ASAN fixed-size allocation size increase, we must always
+           perform two allocations that sum to 64 bytes. */
+        ZSTD_cwksp_reserve_internal(ws, ws->tableExtraAlignmentBytes/2, ZSTD_cwksp_alloc_tables);
```

Comment:
Could this be cleaned up by allocating a 0-sized object when `bytesToAlign == 0`?